discovery-provider/src/queries/health_check.py

```diff
@@ -109,12 +111,15 @@ def health_check():
         # DB connections check
         health_results["db_connections"] = _get_db_conn_state()
 
-    if health_results["block_difference"] > HEALTHY_BLOCK_DIFF:
+    # Return error on unhealthy block diff if requested.
+    enforce_block_diff = request.args.get("enforce_block_diff", type=str) == 'true'
```

Comment:
note - added this flag so I can add a pingdom block check without having to call the /block_check route which makes a web3 call each time

Reply:
honestly i feel like we can just add another flag to health check to force web3 and just eliminate block_check but lgtm